sql_metadb/derived_tables/holding_notes.sql

```diff
@@ -0,0 +1,45 @@
+DROP TABLE IF EXISTS holdings_notes;
+CREATE TABLE holdings_notes AS

```

Comment:
There's a short documentation missing. You might  just copy the brief description from the ldp version:
`-- Create a local table for notes in holdings records that includes the type id and name. Here note can be either public or for staff.`